tests/Xtest_dicom_header.py

```diff
@@ -13,8 +14,8 @@
 import tempfile
 import unittest
 
-from deid.data import get_dataset
 from deid.utils import get_installdir
+from tests.common import get_dataset
```

Comment:
This should ideally be relative so we aren't counting on the pythonpath or present working directory have a folder "tests"

Reply:
I've replaced with `tests_commons` is it ok for you ? Seems like relative imports + unittest don't work that well together :hot_face: